docs/nidigital/class.rst

```diff
@@ -1557,6 +1557,66 @@ export_signal
 
             :type output_terminal: str
 
+fetch_capture_waveform
+----------------------
+
+    .. py:currentmodule:: nidigital.Session
+
+    .. py:method:: fetch_capture_waveform(site_list, waveform_name, samples_to_read, timeout)
```

Comment:
* I thought sites were a repeated capability in which case I expect the signature to be different.
* In most nimi-python drivers we have default timeouts.

If either of these is meant to be fixed separately please reply with a reference to the corresponding issue.

Reply:
I believe sites are like channels in the sense that from Python we should support `"0"`, `"0-1"`, `0`, `[0, 1]`, etc.

Reply:
We don't know what sites will become. They may become rep_caps, but they may not.

Reply:
#1111 opened

Reply:
> In most nimi-python drivers we have default timeouts.

There is a [issue](https://github.com/ni/nimi-python/issues/1072) open to have defaults where it makes sense and make the parameters optional.